torch/testing/_internal/common_methods_invocations.py

```diff
@@ -16296,6 +16296,11 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
                    ),
                    # lgamma have multiple singularities at x <= 0
                    reference_numerics_filter=NumericsFilter(condition=lambda x: x < 0.1, safe_val=1)),
+    OpInfo('multigammaln',
+           dtypes=floating_and_complex_types(),
+           supports_out=False,
+           supports_forward_ad=True,
+           supports_fwgrad_bwgrad=True),
```

Comment:
There are existing OpInfos for `mvlgamma/multigammaln` (So probably, a new entry shouldn't be required)

https://github.com/pytorch/pytorch/blob/ca9919e3e81bb563422beb83e61b71cb8deca62c/torch/testing/_internal/common_methods_invocations.py#L13367-L13386

Reply:
I was getting a no OpInfo found error. Maybe I need to update how I use PythonRefsInfo